queries.sql

```diff
@@ -0,0 +1,27 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * from animals WHERE name = 'Luna';
```

Comment:
[OPTIONAL]
- I assume that this was present when the files were generated. Feel free to remove it as you do not need it. 😃